tools/ExternalObjectReplicator/tests/test_copy_util.py

```diff
@@ -2,7 +2,7 @@
 import boto3
 import datetime
 from unittest.mock import patch, MagicMock
-from unittest import mock
+from unittest import mock, IsolatedAsyncioTestCase
```

Comment:
I am not able to import IsolatedAsyncioTestCase in python 3.7.1
Using this version so sync manager works. ( I assume you run a later version of python and I hope sync manager still works with the newer versions.)

Reply:
It works for 3.8 and above 
https://docs.python.org/3/library/unittest.html